lib/private/Authentication/WebAuthn/Db/PublicKeyCredentialEntity.php

```diff
@@ -41,6 +41,9 @@
  * @method void setPublicKeyCredentialId(string $id);
  * @method string getData();
  * @method void setData(string $data);
+ * @since 30.0.0
```

Comment:
I think this since attribute might be misinterpreted by tools and make it look like the whole class is new in 30

Reply:
Some other files do it, but they have a comment after the _since_
https://github.com/nextcloud/server/blob/master/lib/public/IAppConfig.php
https://github.com/nextcloud/server/blob/master/lib/public/SystemTag/SystemTagsEntityEvent.php
https://github.com/nextcloud/server/blob/master/lib/public/AppFramework/Utility/ITimeFactory.php

Reply:
Maybe you can find out the original since and then add that and a second `@since` with a comment of your changes?

Reply:
So, just adding a comment ?
```suggestion
 * @since 30.0.0 Add userVerification attribute
```

Reply:
Done